.spellcheckwordlist.txt

```diff
@@ -7,6 +7,7 @@ Apache
 Acquia
 amazee.io
 Autocompletion
+AutocompleteTerms
```

Comment:
```suggestion
```

Shouldn't be necessary, was forced by simple error in markdown.

Reply:
Removed